### PR TITLE
Card

### PR DESCRIPTION
**Pull request by amartya-rs**
feature update: card component

card
card-2

### CODE REVIEW

File: utilities/margin.css [hunk 1]
```diff
@@ -0,0 +1,238 @@
+/*m1*/
+.m-1 {
+    margin: 0.5rem;
```
Comment (iRohitGaur) [inline utilities/margin.css:3]: any reason to give in rem?

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by Omkar-Ghate — APPROVED** [review]
Great work! Looks good to merge.

File: utilities/margin.css (hunk 1 above)
Comment (amartya-rs) [reply utilities/margin.css:3]: Every other padding, margin is in rem so I thought of keeping m-1 in rem too.

**Pull request merged**